examples/postgresql-multiple-datasources/pom.xml

```diff
@@ -0,0 +1,79 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wildfly.wildfly-s2i.examples</groupId>
+    <artifactId>postgresql-multiple-datasources</artifactId>
+    <version>1.0</version>
+    <packaging>war</packaging>
+
+    <name>WildFly S2I - Multiple PostgreSQL Datasource </name>
+    <description>An application that connects to multiple PostgreSQL databases</description>
+
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <version.maven.war.plugin>3.3.2</version.maven.war.plugin>
+        <version.wildfly>26.1.1.Final</version.wildfly>
+        <version.wildfly.plugin>4.0.0.Alpha2</version.wildfly.plugin>
```

Comment:
You could use version 4.0.0.Beta3.

Reply:
Updated with latest available version (4.0.0.Beta3)